fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -1842,16 +1917,16 @@ ACTOR Future<SrcDestTeamPair> getSrcDestTeams(DDQueueData* self,
 	return {};
 }
 
-ACTOR Future<Void> BgDDLoadRebalance(DDQueueData* self, int teamCollectionIndex, int ddPriority) {
+ACTOR Future<Void> BgDDLoadRebalance(DDQueueData* self, int teamCollectionIndex, DataMovementReason reason) {
 	state int resetCount = SERVER_KNOBS->DD_REBALANCE_RESET_AMOUNT;
 	state Transaction tr(self->cx);
 	state double lastRead = 0;
 	state bool skipCurrentLoop = false;
 	state Future<Void> delayF = Never();
-	state const bool readRebalance = !isDiskRebalancePriority(ddPriority);
+	state const bool readRebalance = !isDataMovementForDiskBalancing(reason);
```

Comment:
Is `ForDiskBalancing` and `readRebalance` the only two reasons? This may cause confusion if we have more rebalancing reasons in the future, can we add an explicit `isDataMovementForReadBalancing`?